curator-recipes/src/test/java/org/apache/curator/framework/recipes/cache/TestTreeCache.java

```diff
@@ -377,7 +377,6 @@ public void testKilledSession() throws Exception
         assertEvent(TreeCacheEvent.Type.NODE_ADDED, "/test/me");
 
         KillSession.kill(client.getZookeeperClient().getZooKeeper(), server.getConnectString());
-        assertEvent(TreeCacheEvent.Type.CONNECTION_SUSPENDED);
```

Comment:
Why does the CONNECTION_SUSPENDED event no longer occur? Is this because of the new lifecycle stuff for session loss?